### ISSUE TITLE
pbr version capping in detox causing conflicts

### ISSUE THREAD

**Issue by ssbarnea** [tox-dev/tox#792]
I think that this line states it well:
```
detox 0.11 has requirement tox<3.0.0,>=2.0.0, but you'll have tox 3.0.0 which is incompatible.
```

**Comment by RonnyPfannschmidt** [comment]
this is not a tox issue and fixed in detox master

**Comment by ssbarnea** [comment]
@RonnyPfannschmidt issue tracker is closed on detox and it states that we should use tox one instead. Adding detox label cannot be done by contributors.

On the other hand, we could repurpose the ticket into "make a new release of detox" ;)

**Comment by RonnyPfannschmidt** [comment]
oh, i see ^^ i missed that bit, thanks for the update, also note that im no longer a team member ^^

**Comment by obestwalter** [comment]
That‘s a good aim to shoot for.

**Comment by gaborbernat** [comment]
yeah I agree, we should get there via travis somehow :smile:

**Comment by obestwalter** [comment]
oh ... I just saw that I don't have the maintainer bit set for detox yet, so I can't release it. I will get in touch with Holger to do the release or add me as maintainer.

**Comment by obestwalter** [comment]
Got the maintainer bit now and just released 0.12 - it should work smoothely with tox 3 now.
